### PR TITLE
Feat/small website improvements

### PR DESCRIPTION
**Pull request by glecler**
Lots of small changes :

changes to text in en.json and nl.json
added categories to FAQ
added prototype career page
changed form in book a demo
minor cosmetic changes
removed useless images

### CODE REVIEW

File: frontend/public/widget/co2okWidgetMark-d41d8cd9.js [hunk 1]
```diff
@@ -86,16 +86,16 @@ let Co2okWidget = {
...
-        jQuery(".shipping-module").append('<p><i class="icon-earth"></i><br><strong>Shop climate neutral</strong><br>With CO₂ok</p>')
+        jQuery(".shipping-module").append('<p><i class="icon-earth"></i><br><strong>Shop climate neutral</strong><br>With CO2ok</p>')
         jQuery(".icon-earth").css({"color":"#545454"})
       }
 
       if (totalCompensatedData <500) {
         var compensatiewidget  = 137.42;
-        // var compensatietekst = `De bij CO₂ok aangesloten webshops hebben samen <strong> ${compensatiewidget .toFixed(1)} </strong>ton CO₂-uitstoot voorkomen <br><br>= <strong>${(compensatiewidget * 5000).toFixed(0)} </strong>km vliegen`;
+        // var compensatietekst = `De bij CO2ok aangesloten webshops hebben samen <strong> ${compensatiewidget .toFixed(1)} </strong>ton CO₂-uitstoot voorkomen <br><br>= <strong>${(compensatiewidget * 5000).toFixed(0)} </strong>km vliegen`;
       }
```
Comment (TheApeMachine) [inline frontend/public/widget/co2okWidgetMark-d41d8cd9.js:99]: Lol, search and replace eh? No need for now, but let's remove anything commented. Git has the history ;)

File: frontend/src/components/dashboard/Transactions.vue [hunk 2]
```diff
@@ -1,1078 +1,769 @@
...
+					</v-tooltip>
+				</div>
+			</v-flex>
+			<v-flex xs12 sm12 md6 lg6 style="height: 100%;" class="year-info-flex">
+				<!-- <select name="" id="" @change="transactionsDataFormat()" style="width: auto; height: 30px; border: 1px solid #E5E5E5; border-radius: 5px; padding-left: 10px;padding-right: 10px;color: black;font-size: 14px;font-weight: bold;" v-if="!week">
+					<option value="">Choose a data format</option>
+					<option value="transactions">Transactions</option>
+					<option value="compensations">Compensations Sum</option>
+				</select> -->
+				<div class="" style="display: flex; flex-direction:row;justify-content:flex-start;align-items:flex-start;width:auto;cursor:pointermargin-right:20px;">
```
Comment (TheApeMachine) [inline frontend/src/components/dashboard/Transactions.vue:58]: Same, remove comments (but next time, don't bother for now)

File: frontend/src/components/dashboard/Transactions.vue [hunk 3]
```diff
@@ -1,1078 +1,769 @@
...
+					<button class="export__btn" @click="exportPDF()">
+						<v-icon color="white" light style="font-size: 15px; margin-right: 8px;">insert_drive_file</v-icon>PDF
+					</button-->
+					<!-- <button class="export__btn">
+						<v-icon color="white" light style="font-size: 15px; margin-right: 8px;">collections</v-icon>.csv
+					</button> 
+				</div>
+			</div>
+		</div-->
+	</div>
```
Comment (TheApeMachine) [inline frontend/src/components/dashboard/Transactions.vue:100]: Same about the comments.

File: frontend/src/components/dashboard/Transactions.vue [hunk 4]
```diff
@@ -1,1078 +1,769 @@
...
+				//	line: {
+				//		borderWidth: 2,
+				//		backgroundColor: 'rgba(0,0,255,0)',
+				//	},
+				//	point: {
+				//		backgroundColor: 'rgba(148,237,206,0.8)',
+				//		radius: 3,
+				//		hoverRadius: 4
+				//	}
+				//},
```
Comment (TheApeMachine) [inline frontend/src/components/dashboard/Transactions.vue:153]: yep. (comments)

File: frontend/src/components/dashboard/Transactions.vue [hunk 5]
```diff
@@ -1,1078 +1,769 @@
...
+		//	for (let m = 1; m <= months; m++) {
+		//		pdf.push([])
+		//		daysOfMonth = this.$moment(m, 'M').daysInMonth()
+		//		weekOfMonth = Math.ceil(daysOfMonth/7)
+		//		for (let w = 1; w <= daysOfMonth; w++) {
+		//			pdf[m-1].push(w)
+		//		}
+		//		chunkedPDF = this._.chunk(pdf[m],7)
+		//	}
+		//},
```
Comment (TheApeMachine) [inline frontend/src/components/dashboard/Transactions.vue:537]: COM......M.....ENTS :p

**Review by TheApeMachine — APPROVED** [review]
Yep yep, good. Not quite sure why there are so many changes and we are working on the charts here, but I am sure there is a good reason :)

**Comment by glecler** [comment]
All the changes were made before we decided to start over :)

**Comment by TheApeMachine** [comment]
That makes sense :)

**Comment by TheApeMachine** [comment]
Does this need to be deployed or do you know how to do that?

**Pull request merged**